language/Portuguese_Brazil/strings.po

```diff
@@ -200,35 +200,35 @@ msgstr "Adicionar filtro"
 
 #: src/frontend/mame/ui/utils.cpp:950
 msgid "Select category:"
-msgstr ""
+msgstr "Selecione a categoria"
 
 #: src/frontend/mame/ui/utils.cpp:981
 msgid "[no category INI files]"
-msgstr ""
+msgstr "[arquivos INI sem categorias]"
 
 #: src/frontend/mame/ui/utils.cpp:989
 msgid "[no groups in INI file]"
-msgstr ""
+msgstr "[sem grupos no arquivo INI]"
 
 #: src/frontend/mame/ui/utils.cpp:1023
 msgid "No category INI files found"
-msgstr ""
+msgstr "arquivos INI sem categoria encontrada"
```

Comment:
a better wording here could be: "encontrados arquivos INI sem categoria" (INI files lacking category were found) instead of what seems to poorly translate into something like "INI files without found-category"